src/components/Header/web3/WalletModal/index.tsx

```diff
@@ -338,6 +338,9 @@ export default function WalletModal() {
         {(walletView === WALLET_VIEWS.ACCOUNT || walletView === WALLET_VIEWS.CHANGE_WALLET) && (
           <TermAndCondition onClick={() => setIsAcceptedTerm(!isAcceptedTerm)}>
             <input
+              onChange={() => {
```

Comment:
to fix this warning
<img width="566" alt="image" src="https://user-images.githubusercontent.com/33005392/206955548-7d50fb90-b2e1-4b03-9a8a-1f6e5a0dd7bd.png">